pom-jna-platform.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>net.java.dev.jna</groupId>
   <artifactId>jna-platform</artifactId>
-  <version>4.1.0</version>
+  <version>4.1.1-SNAPSHOT</version>
```

Comment:
I/we have our own clone of the repo, and we make an internal release.
This release needs to be the next one on from the latest jna release, so we know we are using a modification.
the trunk (or master) would normally be a SNAPSHOT if it is under development.

Do you have an alternative suggestion of how to handle this!